<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loaddoctor</class>
 <widget class="QDialog" name="loaddoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 117, 143);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>711</width>
     <height>741</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>701</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDocAdd">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>420</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
image: url(:/image/adddoctor.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDocDelete">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>570</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
image: url(:/image/delete.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDocTouch">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>500</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/assesstive.png);
background-color:transparent;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDocLoad">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>630</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>701</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPatAdd">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>450</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
image: url(:/image/addperson.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPatTouch">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>530</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/assesstive.png);
background-color:transparent;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPatDelete">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>600</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
image: url(:/image/delete.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPatLoad">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>620</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
